src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/integration/pact/EndOfServiceReportContracts.kt

```diff
@@ -6,23 +6,27 @@ import java.util.UUID
 
 class EndOfServiceReportContracts(private val setupAssistant: SetupAssistant) {
   @State("There is an existing sent referral with ID of 03bf1369-00d3-4b7f-88b2-da3cc8cc35b9, and it has an end of service report")
-  fun `create referral with eosr`() {
```

Comment:
we did we change this Andrew, is this not supported.

Reply:
Latest version of KTLint throws a violation @kth13 if methods aren't camelCase

Reply:
Is there not a possibility of disabling that rule? The reason is the previous method names were intuitive